docs/typescript/activities.md

```diff
@@ -7,7 +7,7 @@ description: Activities are the only way to interact with external resources in
 
 > **@temporalio/activity** [![NPM](https://img.shields.io/npm/v/@temporalio/activity)](https://www.npmjs.com/package/@temporalio/activity) [API reference](https://typescript.temporal.io/api/namespaces/activity) | [GitHub](https://github.com/temporalio/sdk-typescript/tree/main/packages/activity)
 
-**Activities are the only way to interact with external resources in Temporal**, like making an HTTP request or accessing the file system.
+**Activities are the only way to interact with external resources in Temporal**, such as making an HTTP request or accessing the file system.
```

Comment:
Very minor nit that you could probably easily argue against me on: _"like"_ says there are things similar to the examples; _"such as"_ directly lists examples.

Reply:
I agree with the suggested edit...but because Temporal follows the Google Developer Documentation Style Guide, either term is acceptable. (See "like versus such as" in https://developers.google.com/style/word-list#letter-l.)

Reply:
Aha, thank you for that link! TIL 👍

Reply:
If I'm following my personal style guide, I definitely use "such as" in contexts such as this one. But even the Microsoft Writing Style Guide (which is our backup style guide for issues not covered by the Google guide) now places "like" and "such as" on even terms. (See, for instance, https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/and-so-on.)

Reply:
```suggestion
**Activities are the only way to interact with external resources in Temporal**, such as making an HTTP request or accessing the file system.
```